examples/c/profile.c

```diff
@@ -139,6 +222,12 @@ int main(int argc, char * const argv[])
 		}
 	}
 
+	err = bootstrap_parse_cpu_mask_file(online_cpus_file, &online, &num_online_cpus);
```

Comment:
make sure that if parsing succeeds, you free `online` mask in cleanup

Reply:
done